packages/netlify-cms-core/src/backend.ts

```diff
@@ -90,6 +93,19 @@ const sortByScore = (a: fuzzy.FilterResult<EntryValue>, b: fuzzy.FilterResult<En
   return 0;
 };
 
+export const getFieldsNames = (fields: EntryField[], prefix = '') => {
```

Comment:
I didn't see this function used anywhere.

Reply:
You're right I need to remove it